### PR TITLE
COTR-31-Damage-popup

### PR DESCRIPTION
**Pull request by Arnelys**
Created damage indicator that pops up when player takes damage.

### CODE REVIEW

File: Scenes/UI/DamagePopup/DamagePopup.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                label.Set("theme_override_colors/font_color", damageColor);
+                break;
+        }
+        Random rand = new Random();
+        float sideMovementx = (rand.Next() % 81) - 61;
+        float sideMovementy = (rand.Next() % 81) - 61;
+        velocity = new Vector2(10, 0);
+        tween = CreateTween();
+        tween.TweenProperty(this, "scale", maxSize, 0.7f);
+        tween.Chain().TweenProperty(this, "scale", new Vector2(0.1f, 0.1f), 0.7f).SetDelay(0.3f);
```
Comment (Karolinskis) [inline Scenes/UI/DamagePopup/DamagePopup.cs:41]: comment these parameters for better readability

Comment (Karolinskis) [inline Scenes/UI/DamagePopup/DamagePopup.cs:41]: comment these parameters for better readability

**Review by Karolinskis — CHANGES_REQUESTED** [review]

File: Scenes/UI/DamagePopup/DamagePopup.cs (hunk 1 above)
Comment (Arnelys) [reply Scenes/UI/DamagePopup/DamagePopup.cs]: Done

File: Scenes/UI/DamagePopup/DamagePopup.cs [hunk 2]
```diff
@@ -0,0 +1,61 @@
+using Godot;
+using System;
+
+public partial class DamagePopup : Marker2D
+{
+    // Called when the node enters the scene tree for the first time.
```
[the remark this answers is not in the document]
Comment (Arnelys) [reply Scenes/UI/DamagePopup/DamagePopup.cs]: Done

**Review by Elijuszek — APPROVED** [review]
LGTM!

**Pull request merged**